docs/MachineConfigController.md

```diff
@@ -45,8 +45,16 @@ type MachineConfigPoolSpec struct {
     // This includes generating new desiredMachineConfig and update of machines.
```

Comment:
feel free to punt from this PR, but this could probably use some kind of warning about the risks of long pauses.

Reply:
We have a jira card to document paused pool behavior in 4.10. I think once we have it in place, we can link it down here.